static/scripts/loggedin.js

```diff
@@ -159,7 +161,22 @@ $(document).ready(function () {
 			Es können keine Klassen und Nutzer angelegt werden.
 			Bitte läute <a href="/administration/school/"> hier das neue Schuljahr ein!</a>`, 'warning');
 		}
-	}
+    }
+    
+    // EBS-System | Alert
+	$.ajax({
+		url: '/alerts',
+		contentType: 'application/json',
+		dataType: 'json',
+		success(result) {
```

Comment:
success means only status===200, you should add error handler too (console.error...)

Reply:
and use localstorage for caching